demo/assets/tailwind.config.js

```diff
@@ -7,11 +7,23 @@ module.exports = {
     themes: [
       {
         light: {
-          ...require('daisyui/src/theming/themes').light,
+          ...require('daisyui/src/theming/themes').light
+          ,
           primary: '#1d4ed8',
           'primary-content': 'white',
           secondary: '#f39325',
-          'secondary-content': 'white'
+          'secondary-content': 'white',
+          info: '#dbeafe',
+          'info-content': '#2563eb',
+          'error-content': '#e11d48',
+          'base-200': '#f3f4f6',
+          'base-300': '#e5e7eb'
+        },
+        dark: {
+          ...require('daisyui/src/theming/themes').dark
+        },
+        cyberpunk: {
+          ...require('daisyui/src/theming/themes').cyberpunk
```

Comment:
Did some experiments here, the topbar is remounting everytime a sidebar link is clicked which make the theme selector remount everytime, also, there's the question of persistence, probably will need a controller to save the theme to a session or cookie similarly to what `phx.gen.auth` does and load it as an assign in the root template to avoid loading the default theme and then changing after the mount.

It will be a bit more involved, so I think that another PR is a better place to do it.

Reply:
> It will be a bit more involved, so I think that another PR is a better place to do it.

👍